docs/conf/vilnius/2019/news/announcing-schedule-activities.rst

```diff
@@ -0,0 +1,45 @@
+:template: {{year}}/generic.html
+
+.. post:: May 23, 2019
+   :tags: 2019, vilnius-2019, schedule, tickets
+
+Announcing Full Schedule, Tickets, Events, and Activities
+=========================================================
+
+Hello everyone!
+
+Our first-year event in Vilnius is just over a week away, and we hope everyone is as excited as we are. We're writing today to announce the full presentation schedule and to give you the latest updates to help you make the most out of your conference experience.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the `talk schedule <https://www.writethedocs.org/conf/vilnius/2019/schedule/>`_ for the main stage presentations, so you can start planning your time (and getting excited!) for the conference.
+
+We'll also be running the `Writing Day <https://www.writethedocs.org/conf/vilnius/2019/writing-day/>`_ on Sunday and `lightning talks <https://www.writethedocs.org/conf/vilnius/2019/lightning-talks/>`_ after lunch on Monday.
+
+You'll be able to sign up for Writing Day projects and lightning talks during the conference.
+
+Tickets are going!
+------------------
+
+We still have some tickets available, so head over to the `Tickets <https://www.writethedocs.org/conf/vilnius/2019/tickets/>`_ page and grab one!
+
+Due to catering and printing requirements, we will freeze ticket sales on **Monday, 27th May**. If you register after this date, we will not be able to print a badge for you (although you probably don't need to worry about the food!).
```

Comment:
I take it this means you can't buy a ticket anymore, but you can still register it to your name, after this date? Or not? I am confused by the combination of "freeze sales" with "register after this date".